install.sh

```diff
@@ -76,6 +76,9 @@ chmod +x ${CSFG}
 # would duplicate warnings provided by ublue already. we don't want confusion
 rm -f /usr/libexec/gnome-software-dkms-helper
 
+# Remove avif thumbnailer, as heif thumbnailer already caches it
```

Comment:
Is there a negative to the current state?

Do both thumbnailers run, thus wasting resources?
If not...
Does one always get chosen instead of the other, or is it random?

Reply:
Nothing breaking, thumbnailing still works.

I'm not sure if both thumbnailers are running at the same time.

I think that one gets chosen over the other, but on which terms, I don't know.